src/Plugin/Field/FieldFormatter/StrawberryMetadataTwigFormatter.php

```diff
@@ -152,25 +152,31 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
     }
 
     return [
-      'label' => [
-        '#type' => 'string',
-        '#title' => $this->t('Public facing Label for this Metadata Display'),
-        '#default_value' => $this->getSetting('label'),
+      'cutomtext' => [
```

Comment:
@marlo-longley can you change this type? Should say "customtext" of "infotext" not cutomtext.

Reply:
@marlo-longley ping. This has been around quite long without being addressed. Can you please on monday fix these? thanks